### PR TITLE
Ens avatar support

### PR DESCRIPTION
**Pull request by sasicodes**
- Shows ens avatar only if the user is already set. For users with no avatar, it won't be displayed.

No libraries installed.

image

### CODE REVIEW

**Comment by sasicodes** [comment]
@fritzschoff Can you please review this 🙇🏻

File: sections/shared/Layout/UserMenu/UserMenu.tsx [hunk 1]
```diff
@@ -219,8 +227,16 @@ const Menu = styled.div`
 	grid-auto-flow: column;
 `;
 
-const EnsName = styled.span`
+const Ens = styled.span`
+	display: flex;
+	align-items: center;
 	text-transform: lowercase;
+	gap: 5px;
```
Comment (fritzschoff) [inline sections/shared/Layout/UserMenu/UserMenu.tsx:234]: unfortunately gap is not supported in safari when using flexbox. 
There are not many users that connect via safari to our dapp but still, would be annoying for them. 
Can you make the spacing with margin? Or something else please ?

**Review by fritzschoff — CHANGES_REQUESTED** [review]

**Comment by fritzschoff** [comment]
@sasicodes thank very much! Small minor issue. After that we are good to go :)

File: sections/shared/Layout/UserMenu/UserMenu.tsx (hunk 1 above)
Comment (sasicodes) [reply sections/shared/Layout/UserMenu/UserMenu.tsx:234]: sure

**Comment by sasicodes** [comment]
@fritzschoff Updated ser!

**Comment by fritzschoff** [comment]
Awesome. Thanks so much!

**Pull request merged**